frontend/src/App.test.tsx

```diff
@@ -222,10 +222,15 @@ describe("App.handleNewReport", () => {
   afterEach(() => {
     const UnsafeSessionInfo = SessionInfo as any
     UnsafeSessionInfo.singleton = undefined
+    window.localStorage.clear()
   })
 
   it("adds custom theme to list of available themes", () => {
     const props = getProps()
+    window.localStorage.setItem(
```

Comment:
nit: there's a skipped test with a TODO down below that this test covers the behavior for now